web/application/views/airqo-blog.php

```diff
@@ -1,5 +1,5 @@
 <style>
-.slider-post[data-v-5e6923b3] {
+/* .grid-post[data-v-5e6923b3] {
     position: relative;
```

Comment:
Thanks, awesome stuff @danielogen 
Could we remove these comments?
Unless we plan to use them in the near future?

Reply:
resolved